CHANGELOG.md

```diff
@@ -7,7 +7,7 @@
     - Update to version 0.1.1 of lauramurgatroyd/build-sphinx-action for building the documentation - ensures docs are always built from cil master.
 
 * 21.4.1
- - Strongly convex functionality in TotalVariation and FGP_TV Functions
+ - Strongly convex functionality in TotalVariation and FGP_TV Functions.
```

Comment:
This is a 22.0.0 change, not 21.4.1. Move it up.

Reply:
Fixed